tools/gen.py

```diff
@@ -16,11 +15,17 @@ def factory():
 
 
 TERMINAL = object()
+HIDDEN_FROM_FIND_HEADER = (
+    'Proxy-Authorization',
```

Comment:
Please elaborate why do you want to skip the header

Reply:
Ah, thanks.
I don't recall details but IMHO there is no reason for `Proxy-Authorization` header skipping.

Please revert skipping logic in `gen.py` and regenerate `_find_headers.c`

Reply:
done